pkgs/by-name/od/odri-masterboard/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  fetchFromGitHub,
+  fetchgit,
+  stdenv,
+  cmake,
+  yaml-cpp,
+  eigen,
```

Comment:
should probably yeet that, i cp'ed from the other package.nix

Reply:
you can run deadnix to automatically yeet what is obviously yeetable